README.md

```diff
@@ -1,208 +1,143 @@
-# ForgeRock JavaScript SDK
+[![npm (scoped)](https://img.shields.io/npm/v/@forgerock/javascript-sdk?color=%23f46200&label=Version&style=flat-square)](CHANGELOG.md)
 
-The ForgeRock JavaScript SDK is a toolkit that allows web developers to easily integrate intelligent authentication using ForgeRock OpenAM and/or ForgeRock Identity Cloud.
+
+<p align="center">
+  <a href="https://github.com/ForgeRock">
+    <img src="https://www.forgerock.com/themes/custom/forgerock/images/fr-logo-horz-color.svg" alt="Logo">
+  </a>
+  <h2 align="center">ForgeRock SDK for JavaScript</h2>
+  <p align="center">
+    <a href="./blob/master/CHANGELOG.md">Change Log</a>
+    ·
+    <a href="#support">Support</a>
+    ·
+    <a href="#documentation" target="_blank">Docs</a>
+  </p>
+  <hr/>
+</p>
+
+The ForgeRock JavaScript SDK enables you to quickly integrate the [ForgeRock Identity Platform](https://www.forgerock.com/digital-identity-and-access-management-platform) into your JavaScript apps.
```

Comment:
Should we say "client-side JavaScript apps"? That may seem silly, but it is a distinction that makes a difference.

Reply:
Doesn't hurt to be specific; I've added client-side.